website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -1,70 +1,81 @@
 
-_Notice: the `Interactive` list item component internally uses the generic `Hds::Interactive` component. For more details about how this low-level component works please refer to [its documentation page](/utilities/interactive/)._
+<!-- TODO This breaks the page...why?
+The `Interactive` list item renders the correct element based on the passing of an `@route`, `@href`, or the addition of a click event (i.e.,
+`{{on "click" this.myAction}}`). 
+-->
 
-##### Basic use
+Internally, the component uses the [Hds::Interactive](/utilities/interactive/) utility component.
 
-If you don't pass a `@href` or `@route` argument a simple `<button>` will be generated:
+#### Rendering a button
+
+If you add an event handler (no `@href` or `@route`), a `<button>` element will be rendered:
 
 ```handlebars
-<Hds::Dropdown as |dd|>
-  ...
-  <dd.Interactive {{on "click" this.myAction}} @text="Run command" />
-</Hds::Dropdown>
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive {{on "click" this.myAction}} @text="Run command" />
+</Doc::ListContainer>
 ```
 
-_Notice: in this case you will have to add your own event handling function to it._
+#### Rendering a link with `@href`
 
-##### With @href
-
-If you pass a `@href` argument a `<a>` link will be generated:
+If you pass a `@href` argument, a link (`<a>` element) will be generated:
 
 ```handlebars
-<Hds::Dropdown as |dd|>
-  ...
-  <dd.Interactive @href="https://www.hashicorp.com/request-demo/terraform" @text="Request a demo" />
-</Hds::Dropdown>
+{{!-- The Doc::ListContainer component is just to help the component render properly --}}
+<Doc::ListContainer class="hds-dropdown-list">
+  <Hds::Dropdown::ListItem::Interactive @href="https://www.hashicorp.com/request-demo/terraform" @text="Request a demo" />
+</Doc::ListContainer>
 ```
 
-**Important**: when using the `@href` argument the component adds by default the attributes `target="_blank"` and `rel="noopener noreferrer"` to the `<a>` element (because this is the most common use case: internal links are generally handled using a `@route` argument). If the `href` points to an internal link, or uses a different protocol (eg. "mailto" of "ftp") you can pass `@isHrefExternal={{true}}` to the component and it will not add the `target` and `rel` attributes (but you can pass yours if needed, using the `...attributes` spreading. For more details see the [Hds::Interactive component](/utilities/interactive/).
+!!! Info
+
+**Internal Link?**
+
+When using the `@href` argument, you’re indicating an external link (instead of a route). So, a few relevant HTML attributes are added–`target="_blank"` and `rel="noopener noreferrer"`. However, if the `@href` really _does_ point to an internal link or uses a different protocol (e.g., `mailto` or `ftp`), pass `@isHrefExternal={{true}}` to the component and it will not add any extra HTML attributes.
+
```

Comment:
Shouldn't this be "false"? `@isHrefExternal={{true}}`

Reply:
good catch!